ads/weborama.md

```diff
@@ -0,0 +1,95 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Weborama
```

Comment:
Unfortunately, I think that we only have internal documentation that explains our tags. We usually configure the tags for our clients ourselves before sending them out.

I didn't see links to documentation in most of the other ad type definitions. Is it OK if we skip this step too?

Or were you referring to the copyright banner at the top of the file?